vyper/venom/analysis/dfg.py

```diff
@@ -22,6 +22,11 @@ def get_uses(self, op: IRVariable) -> list[IRInstruction]:
     def get_producing_instruction(self, op: IRVariable) -> Optional[IRInstruction]:
         return self._dfg_outputs.get(op)
 
+    def remove_use(self, op: IRVariable, inst: IRInstruction) -> list[IRInstruction]:
+        uses = self._dfg_inputs.get(op, [])
+        uses.remove(inst)
+        return uses
```

Comment:
nit: i'd prefer an API which does one thing at a time, so `remove_use()` just removes the item, and then the caller needs another call to `.get_uses()`

Reply:
It’s a pattern I use regularly, but I have no strong preference about doing it in two steps